tests/functional/spec/features/routing/boolean.spec.js

```diff
@@ -0,0 +1,22 @@
+import Block1Page from "../../../generated_pages/metadata_routing/block1.page";
+import Block2Page from "../../../generated_pages/metadata_routing/block2.page";
+import Block3Page from "../../../generated_pages/metadata_routing/block3.page";
+import { click } from "../../../helpers";
+
+describe("Feature: Routing - Boolean Flag", () => {
+  it("Given Boolean Flag is False, When I press continue, Then I should be taken to the Block2Page", async () => {
```

Comment:
The descriptions for tests like this should be generic, and could be made more meaningful. So needs to be reworded so that it describes the behaviour of the feature i.e `Given I have a routing rule that uses a boolean flag` etc

Reply:
Updated the `it` function descriptions